### PR TITLE
Table formatting done for issue #8624

### PR DESCRIPTION
**Pull request by jain-shivangi**
Address the issue in  - [BUG] Table not formatted correctly in docs #8624.
Converted to HTML table as javadoc could not properly display the table.

### CODE REVIEW

File: sdk/identity/azure-identity/README.md [hunk 1]
```diff
@@ -120,16 +120,61 @@ A credential is a class which contains or can obtain the data needed for a servi
...
-|-|-|-
-|`DefaultAzureCredential`|service principal or managed identity|none for managed identity; [environment variables](#environment-variables) for service principal
-|`ManagedIdentityCredential`|managed identity|`ManagedIdentityCredentialBuilder`
-|`EnvironmentCredential`|service principal|[environment variables](#environment-variables)
-|`ClientSecretCredential`|service principal|`ClientSecretCredentialBuilder`
-|`ClientCertificateCredential`|service principal|`ClientCertificateCredentialBuilder`
-|`DeviceCodeCredential`|user account|`DeviceCodeCredentialBuilder`
-|`InteractiveBrowserCredential`|user account|`InteractiveBrowserCredentialBuilder`
-|`UsernamePasswordCredential`|user account|`UsernamePasswordCredentialBuilder`
+<html>
```
Comment (jongio) [inline sdk/identity/azure-identity/README.md:123]: Don't need html, head tags

File: sdk/identity/azure-identity/README.md [hunk 2]
```diff
@@ -120,16 +120,61 @@ A credential is a class which contains or can obtain the data needed for a servi
...
+<head>
+<table style="border-collapse: collapse;">
+<thead>
+<tr>
+<th style="height:40px; width:200px; border:1px solid #dddddd;">credential class</th>
```
Comment (jongio) [inline sdk/identity/azure-identity/README.md:128]: Please use css via a style block.

Comment (jain-shivangi) [reply sdk/identity/azure-identity/README.md:128]: I  tried this but I am not able to use <style> tag inside Markdown correctly. It is rendering <style> block as is, like normal text. Can you please let me know if I am doing something wrong?

Reference- 
https://stackoverflow.com/questions/20598628/do-style-tags-work-in-markdown

File: sdk/identity/azure-identity/README.md (hunk 1 above)
Comment (jain-shivangi) [reply sdk/identity/azure-identity/README.md]: Thanks.  Removed head, html tags.

File: sdk/identity/azure-identity/README.md (hunk 2 above)
Comment (jongio) [reply sdk/identity/azure-identity/README.md:129]: You can just remove all of the CSS and just use the html border property.

Comment (jain-shivangi) [reply sdk/identity/azure-identity/README.md]: Thanks for the help. Done as suggested.

File: sdk/identity/azure-identity/README.md [hunk 3]
```diff
@@ -120,16 +120,62 @@ A credential is a class which contains or can obtain the data needed for a servi
...
-|`EnvironmentCredential`|service principal|[environment variables](#environment-variables)
-|`ClientSecretCredential`|service principal|`ClientSecretCredentialBuilder`
-|`ClientCertificateCredential`|service principal|`ClientCertificateCredentialBuilder`
-|`DeviceCodeCredential`|user account|`DeviceCodeCredentialBuilder`
-|`InteractiveBrowserCredential`|user account|`InteractiveBrowserCredentialBuilder`
-|`UsernamePasswordCredential`|user account|`UsernamePasswordCredentialBuilder`
+
+
+        
+<table border="1">
```
Comment (jongio) [inline sdk/identity/azure-identity/README.md:126]: Please properly indent you can use VS Code for that.

File: sdk/identity/azure-identity/README.md [hunk 4]
```diff
@@ -120,63 +120,64 @@ A credential is a class which contains or can obtain the data needed for a servi
...
+      <th>identity</th>
+      <th>configuration</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>DefaultAzureCredential</code></td>
+      <td>service principal or managed identity</td>
+      <td>
+        none for managed identity;
```
Comment (jongio) [inline sdk/identity/azure-identity/README.md:137]: the formatting is odd here. seems like a one liner would do.

File: sdk/identity/azure-identity/README.md [hunk 5]
```diff
@@ -120,63 +120,64 @@ A credential is a class which contains or can obtain the data needed for a servi
...
+        <a href="#environment-variables">environment variables</a> for service
```
Comment (jongio) [inline sdk/identity/azure-identity/README.md:138]: same, don't need to wrap, i think

File: sdk/identity/azure-identity/README.md [hunk 6]
```diff
@@ -178,6 +174,7 @@ The credential types in Azure Identity differ in the types of AAD identities the
 </table>
 
 
+
```
Comment (jongio) [inline sdk/identity/azure-identity/README.md:177]: Remove all extra lines.  There should be at most one empty line in between sections.

Comment (jongio) [reply sdk/identity/azure-identity/README.md:177]: That applies to above the table as well.

Comment (jain-shivangi) [reply sdk/identity/azure-identity/README.md]: Thank you very much for patiently reviewing and suggesting changes. Will keep these things in mind henceforth.

**Comment by jain-shivangi** [comment]
The check enforcer has been running since a day I think. Is it possible to re trigger it ?

**Comment by jongio** [comment]
The java CI also failing.  Likely something @srnagar can help with.

**Comment by jain-shivangi** [comment]
Thanks for your help and comment. 
All the checks have passed now.

**Comment by jain-shivangi** [comment]
Kindly merge it whenever free.

**Comment by jain-shivangi** [comment]
Thanks a lot, @g2vinay :)

**Pull request merged**